tests/Feature/TeamInvitationEmailNormalizationTest.php

```
@@ -0,0 +1,63 @@
+        $invitation = TeamInvitation::create([
+            'team_id' => $team->id,
+            'uuid' => 'test-uuid-123',
+            'email' => 'Test@Example.com', // Mixed case
+            'role' => 'member',
+            'link' => 'https://example.com/invite/test-uuid-123',
+            'via' => 'link'
+        ]);
+
+        // Verify email was normalized to lowercase
+        $this->assertEquals('test@example.com', $invitation->email);
+    }
```

PHPMD (2.15.0)

The method test_team_invitation_normalizes_email_to_lowercase is not named in camelCase. (undefined)

(CamelCaseMethodName)